templates/case-studies/index.html

```diff
@@ -4,15 +4,27 @@
 
 {% block content %}
   <!--This index page will be using macros to build an example case study -->
-  <!-- Hero section-->
+  
+  <!-- Hero section-->  
   {% import "case-studies/templates/_macros-hero.jinja" as hero_template %}
```

Comment:
We have been importing our macros at the top of the templates that use them, like this:

https://github.com/canonical/vanilla-framework/blob/main/templates/docs/examples/patterns/hero/hero-50-50.html